cob_generic_can/common/include/cob_generic_can/CanItf.h

```diff
@@ -82,10 +83,11 @@ class CanItf
 public:
 	enum CanItfType {
 		CAN_PEAK = 0,
-		CAN_PEAK_USN = 1,
+		CAN_PEAK_USB = 1,
```

Comment:
I see that this has been a type before...but has it been used with `CAN_PEAK_USN` before somewhere? It needs to be changed there too then!